docs/docs/tutorials/orca-yarn-tutorial.md

```diff
@@ -0,0 +1,532 @@
+```bash
+${SPARK_HOME}/bin/spark-submit \
+    --master yarn \
+    --deploy-mode client \
+    --executor-memory 10g \
+    --driver-memory 10g \
+    --executor-cores 8 \
+    --num-executors 2 \
+    --archives /path/to/environment.tar.gz#environment \
+    --properties-file ${BIGDL_HOME}/conf/spark-bigdl.conf \
+    --py-files ${BIGDL_HOME}/python/bigdl-spark_${SPARK_VERSION}-${BIGDL_VERSION}-python-api.zip,model.py \
+    --conf spark.pyspark.driver.python=/path/to/python \
+    --conf spark.pyspark.python=environment/bin/python \
+    --conf spark.driver.extraClassPath=${BIGDL_HOME}/jars/* \
+    --conf spark.executor.extraClassPath=${BIGDL_HOME}/jars/* \
+    train.py --cluster_mode spark-submit --remote_dir hdfs://url:port/path/to/remote/data
+```
+In the `spark-submit` script:
+* `--master`: the spark master, set it to `yarn`;
+* `--deploy-mode`: set it to `client` when running programs on yarn-client mode;
+* `--executor-memory`: set the memory for each executor;
+* `--driver-memory`: set the memory for the driver node;
+* `--executor-cores`: set the cores number for each executor;
+* `--num_executors`: set the number of executors;
+* `--archives`: upload the Conda archive to YARN;
+* `--properties-file`: upload the BigDL configuration properties to YARN;
+* `--py-files`: upload extra Python dependency files to YARN;
+* `--conf spark.pyspark.driver.python`: set the Python location in Conda archive as driver's Python environment;
+* `--conf spark.pyspark.python`: set the Python location in Conda archive as executors' Python environment;
+* `--conf spark.driver.extraClassPath`: upload and register the BigDL jars files to the driver's classpath;
+* `--conf spark.executor.extraClassPath`: upload and register the BigDL jars files to the executors' classpath;
+
+__Note:__
+* `--cluster_mode`: set the cluster_mode in `init_orca_context`;
+* `--remote_dir`: directory on a distributed storage for the dataset (see __Section 3__) and saving the model.
+* Please refer to __Section 4__ for the description of extra Python dependencies.
+* For CDH Users, please speficy `ARROW_LIBHDFS_DIR` (see __Section 2.3__) as a Spark conf in the script.
+    ```bash
+    ${SPARK_HOME}/bin/spark-submit \
+    ...
+    --conf spark.executorEnv.ARROW_LIBHDFS_DIR=/opt/cloudera/parcels/CDH-${CHD_VERSION}/lib64/ \
+    train.py --cluster_mode spark-submit --remote_dir hdfs://url:port/path/to/remote/data
+    ```
+
+### 5.3.2 Yarn-Cluster
+On the __Client Node__ (where you submit applications), please download jars files from [BigDL Dllib Jars](https://repo1.maven.org/maven2/com/intel/analytics/bigdl/bigdl-dllib-spark_2.4.6/2.0.0/bigdl-dllib-spark_2.4.6-2.0.0-jar-with-dependencies.jar) and [BigDL Orca Jars](https://repo1.maven.org/maven2/com/intel/analytics/bigdl/bigdl-orca-spark_2.4.6/2.0.0/bigdl-orca-spark_2.4.6-2.0.0-jar-with-dependencies.jar) separately, and put them under `${BIGDL_HOME}`.
```

Comment:
We need to fix this; it will be very confusing for the user to download jars from different places for different use cases.

Reply:
Sure, it will be fixed soon.

Reply:
@glorysdj @Le-Zheng Can we put these jars in our assembly as well?

Reply:
Fix in https://github.com/intel-analytics/BigDL/pull/5416

Reply:
Will it be slow if we include all the jars in assembly when submitting the job?